flink-contrib/flink-storm-examples/src/main/java/org/apache/flink/storm/util/BoltFileSink.java

```diff
@@ -18,20 +18,23 @@
 package org.apache.flink.storm.util;
 
 import backtype.storm.task.TopologyContext;
+import org.apache.flink.core.fs.FSDataOutputStream;
+import org.apache.flink.core.fs.FileSystem;
+import org.apache.flink.core.fs.Path;
 
 import java.io.BufferedWriter;
-import java.io.FileWriter;
 import java.io.IOException;
+import java.io.OutputStreamWriter;
 import java.util.Map;
 
 /**
- * Implements a sink that write the received data to the given file (as a result of {@code Object.toString()} for each
+ * Implements a sink that writes the received data to the given file (as a result of {@code Object.toString()} for each
  * attribute).
  */
 public final class BoltFileSink extends AbstractBoltSink {
 	private static final long serialVersionUID = 2014027288631273666L;
 
-	private final String path;
+	private final Path path;
 	private BufferedWriter writer;
```

Comment:
That's not the point. If I re-use an existing bolt, I don't want to change anything. And I want to be able (from an example point of view) to run this bolt as-is in Storm, too. (Without any dependencies to Flink)